src/server/routes/entity/publisher.ts

```diff
@@ -207,18 +207,18 @@ router.get('/:bbid', middleware.loadEntityRelationships, middleware.loadWikipedi
 		'editionFormat',
 		'authorCredit.names'
 	];
-	const editionsPromise =
-		Publisher.forge({bbid: res.locals.entity.bbid})
-			.editions({withRelated: editionRelationsToFetch});
 
-	return editionsPromise
-		.then((editions) => {
-			res.locals.entity.editions = editions.toJSON();
-			_setPublisherTitle(res);
-			res.locals.entity.editions.sort(entityRoutes.compareEntitiesByDate);
-			return entityRoutes.displayEntity(req, res);
-		})
-		.catch(next);
+	try {
+		const editions = await Publisher.forge({bbid: res.locals.entity.bbid})
+			.editions({withRelated: editionRelationsToFetch});
+		res.locals.entity.editions = editions.toJSON();
+		res.locals.entity.editions.sort(entityRoutes.compareEntitiesByDate);
+		return await entityRoutes.displayEntity(req, res);
+	}
+	catch (error) {
+		log.debug(error);
+		return next(error);
```

Comment:
You've removed `next` from the arguments at the beginning of this function, this will throw an error.